app/services/import_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ImportHelpers
```

Comment:
We can move it to `app/services/concerns`. I'm not sure if that would require to add new `concern` directory to autoload_paths in `config/application.rb`, like `config.autoload_paths += Dir[Rails.root.join("app", "services", "concerns")]`

Reply:
I think concerns are meant to encapsulate a piece of domain logic, here we have 2 simple helper methods related to different things.

Maybe I add 2 different modules, which could be used like this:

```ruby
def normalize_date
  Import::Date.try_to_parse(date)
end

def attributes
  { location: Import::Location.find(..) }
end

```

Reply:
Also, I'd move all import classes to `services/import` as well, so we'll end up with `Import::Companies.new.call` and so on.

Reply:
Ok.

Reply:
done, please re-review